src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -555,4 +555,10 @@
   <data name="net_quic_streamaborted" xml:space="preserve">
     <value>Stream aborted by peer ({0}).</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="net_http_missing_sync_implementation" xml:space="preserve">
+    <value>The synchronous method is not supported by '{0}'. If you're using a custom '{1}' and wish to use synchronous HTTP methods, you must override its '{2}' virtual method.</value>
+  </data>
+  <data name="net_http_sync_not_supported" xml:space="preserve">
```

Comment:
Nit: can you include HTTP2 in the resource name?  It'll make the call site a bit clearer why this message is being used.